Header header
string parentNode
string errorKey
string errorDescription
uint8 errorLevel